src/components/box/box.test.js

```diff
@@ -53,6 +53,23 @@ const BoxComponent = ({ ...props }) => {
   );
 };
 
+const BoxComponentForPosition = ({ ...props }) => {
+  return (
+    <Box
+      m={3}
+      p={3}
+      width={400}
+      height={400}
+      data-element="box"
+      bg="primary"
+      color="red"
+      {...props}
+    >
+      This is some sample text
+    </Box>
+  );
+};
+
```

Comment:
If you remove the flex properties from the `Box` in `BoxComponent`, all the tests should still pass. So I think you could use this instead of creating a new test component to avoid duplication.

Reply:
I'll give it a shot. If it breaks, I know who to blame😆